pom.xml

```diff
@@ -758,6 +759,11 @@
                 <artifactId>aliyun-sdk-datahub</artifactId>
                 <version>${datahub.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Do not add connector jar dependency in root pom.xml, just add it in your connector's pom.xml. To reduce denpendency conflict.

Reply:
@Hisoka-X Thank you for your reminder, done